testing/ansible-llm/clusters.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+set -o pipefail
+set -o errexit
+set -o nounset
+set -o errtrace
+set -x
+
+TESTING_PIPELINES_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+TESTING_UTILS_DIR="$TESTING_PIPELINES_DIR/../utils"
+
+source "$TESTING_PIPELINES_DIR/configure.sh"
```

Comment:
🤔 you shouldn't reuse `"$TESTING_PIPELINES_DIR/configure.sh"`

or is it WIP, for the cluster creation to work 🤔 

on purpose, I duplicated everything from `notebooks` to `pipelines` so that both tests are 100% independent (the configuration and entrypoints at least), to avoid side effects between one component and the other. 

we can keep this file as is if WIP, no problem

Reply:
This file should have been deleted, good catch